Description: fix errors from pod2man (specify encoding as latin-1)
Author: Florian Schlichting <fsfs@debian.org>
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=105738

--- a/lib/Date/Calc.pod
+++ b/lib/Date/Calc.pod
@@ -1,4 +1,6 @@
 
+=encoding iso-8859-1
+
 =head1 NAME
 
 Date::Calc - Gregorian calendar date calculations
